app/experimenter/experiments/forms.py

```diff
@@ -531,6 +531,16 @@ class Meta:
 
 
 class ExperimentResultsForm(ChangeLogMixin, forms.ModelForm):
+    RADIO_OPTIONS = ((False, "No"), (True, "Yes"))
+
+    def coerce_truthy(value):
+        if type(value) == bool:
+            return value
+        if value.lower() == "true":
```

Comment:
@jaredlockhart The initital two "no"s you were seeing were coming from the factory, where I had prepopulated two results questions fields. The reason I was doing that was because I was not reaching 100% test coverage due to the `coerce_truthy` method for the radio buttons. I wasn't able to hit line 537-538 here. In the risks tests, this line was hit because the factory prepopulates the risk questions as False. But I just realized that when I create the experiment with the factory in the test, if I set a value for a question to False when creating the experiment with the factory, the same thing will be achieved and line 537-538 will be covered. I'm not sure I have explained this well.

Reply:
So you shouldn't see the nos anymore, I took that out of the factory.